schemas/resultSchema.js

```diff
@@ -3,9 +3,7 @@ import * as yup from 'yup';
 export const resultSchema = yup.object().shape({
   _id: yup.string().optional(),
   description: yup.string().required(),
-  // POCApproval: yup.string().matches(/^\d+$/),
-  // SEApproval: yup.string().matches(/^\d+$/),
-  evidence: yup.string().optional(),
+  evidence: yup.string().required(),
```

Comment:
I think users should have freedom to decide if they want to provide evidence. I think this should be optional.

Reply:
I will change it back to optional (may change back later). It is more work to correctly implement optional fields, so unless all of that work has already been done, its probably better to leave as required.